### DISCUSSION TITLE
Packer with Vue 3 and Tailwind

### DISCUSSION THREAD

**Discussion by molotow11** [hayes0724/shopify-packer#108] [Q&A]
Hello, Everybody!

I was unable to setup Packer with Vue 3 and vue-loader. Webpack issues with loading modules etc..
Are you planning to add some demo repo with Vue 3 and Tailwind configuration? Maybe you can help with it?

**Comment by hayes0724** [comment]
It's not currently planned but Tailwind + Vue is one of my favorite combos so I will do some testing and see if I can make a template.

**Reply by molotow11** [reply]
Hello, Eric.
Thank you so much for your answer. It would be very great to have Packer with this combo.
